CHANGELOG.md

```diff
@@ -17,6 +17,8 @@ For older changes see the [archived Singularity change log](https://github.com/a
   older CPUs, although it can have a cost in performance on newer CPUs.
   It is still possible to set `GOAMD64` to a newer microarchitecture (v2+).
   For instance RHEL 9 uses v2 and RHEL 10 uses v3 as their default values.
+- Refine the error message of container creation failure when apptainer-suid
```

Comment:
A better message for this fix would be:
```
- Add a clear error message if someone tries to use privileged network options
  while not using setuid mode.
```
However I don't think this is the correct fix, see below.